docs/CustomWidgetsSDK.md

```diff
@@ -0,0 +1,98 @@
+# Custom Widget SDK Overview
+
+The custom widget SDK exposes functionality from the Cvent platform to the custom elements that make up a custom widget and it's corresponding editor element. These functions are defined on the CE class for the widget and editor elements.
+
+# Methods
+
+## `getEventTitle()`
+
+Returns the title of the event
+| Async? |
+| ---------------------- |
+|yes|
+
+|Parameters | Description       |
+|-------------|-------------------|
+|N/A| No parameters are supplied to this method |
+
+| Return Type | Description       |
+|-------------|-------------------|
+| `string`| The title of the event |
+
+### Example Usage
+
+```javascript
+const title = await getEventTitle()
+```
+
+---------------------------------------------------------
+
+## `getSessionDetails()`
+
+Returns basic information about all of the sessions in the event.
+
+| Async? |
+| ---------------------- |
+|yes|
+
+|Parameters | Description       |
+|-------------|-------------------|
+|N/A| No parameters are supplied to this method |
+
+| Return Type | Description       |
+|-------------|-------------------|
+| `{ SessionId: SessionDetail }`| A dictionary with an entry for each session. A unique UUID for each session is used as a key and associated with a SessionDetail object|
+
+### The Session Detail Object
+
+Conforms to the following typescript type:
+
+```typescript
+type SessionDetail = {
+    // Unique identifier for session
+    id: string;
+    // Name/Title of the session
+    name: string;
+    // Code for session
+    code: string;
+    // Is this session optional or included?
+    isIncludedSession: boolean;
+    // Zoned datetime string for session start
+    startDateTime: Date;
+    // Zoned datetime string for session end
+    endDateTime: Date;
+    // Plain text description of the session
+    description: string;
+    // Object describing the session location
```

Comment:
It's weird that category is an optional field with required properties and location is a required field with optional properties. We should make that consistent.

Reply:
I agree, I'll revisit this in the work I have left to releases the getSessionDetails SDK method.